community/kernel/src/main/java/org/neo4j/kernel/impl/api/store/StoreAbstractNodeCursor.java

```diff
@@ -234,7 +239,7 @@ public boolean next()
                         return false;
                     }
 
-                    relationshipGroupStore.getRecord( groupId, group, NORMAL );
+                    cursors.relationshipGroup().next( groupId, group, NORMAL );
```

Comment:
`RecordCursor` can return `false` only when read record is not in use. After reading, each record is verified by the `LoadMode`. `LoadMode.NORMAL` throws `InvalidRecordException` if record is not in use. So here and in all other places where we read with `NORMAL` check of `next()` result could be skipped because mode would throw if unused record was read.